### ISSUE TITLE
NestedTabularInline drag-and-drop?

### ISSUE THREAD

**Issue by globophobe** [theatlantic/django-nested-admin#97]
Thanks for the great library! NestedStackedInline drag-and-drop works great. 

NestedTabularInline drag-and-drop doesn't seem to work. The first level is fine. However, subsequent levels of nesting don't respond.

**Comment by fdintino** [comment]
Hmm. We have extensive unit tests for drag-and-drop on `NestedTabularInline`, and those run on all django versions from 1.8 to 2.0. Is there anything unusual about your django admin setup? Additional admin extensions, ModelAdmin class mixins, or custom templates?

**Comment by globophobe** [comment]
I disabled everything possible to see if there was a conflict. There are no custom templates. Django==2.0.4

However, I have a base class:
```
class TabularBase(nested_admin.NestedTabularInline):
    sortable_field_name = 'order'
    extra = 0
```

The problem seems to be with **extra = 0**. When **extra = 1** (or greater), drag-and-drop works normally.

**Comment by fdintino** [comment]
@globophobe can you check if the [dj20-fix-tabular-inlines](https://github.com/theatlantic/django-nested-admin/tree/dj20-fix-tabular-inlines) fixes your issue?

**Comment by globophobe** [comment]
There was no change. There were no errors in the javascript console.

I tried a few browsers, and cleared the cache. I had cloned the branch, and installed with "python setup.py develop" 

pip freeze:

`-e git+https://github.com/theatlantic/django-nested-admin.git@3a75b938ca0b802c180df822d2cdc5e58b321aa1#egg=django_nested_admin`

I looked at https://github.com/theatlantic/django-nested-admin/commit/58b4c98f8aa36a7f86f5bfe18fec6494b9f63dfe

However, I'm not familiar with the code base. 

Maybe, I'll look more on the weekend.
